### DISCUSSION TITLE
The default wavelength range to compute the AGN fraction

### DISCUSSION THREAD

**Discussion by W-Chang-astro** [mboquien/cigale#40] [Q&A]
Hi, 

I am curious about the wavelength range for CIGALAE to compute the AGN fraction. Your team answered a question on this discussion page and mentioned the entire IR(or dust) wavelength range (3-1000 um) by default (lambda_fracAGN = 0/0). Is there a published reference from your team that includes this choice? Since the different decisions of range might influence AGN fraction, would you mind explaining a little more about why you chose this range? Thanks!

---- From CIGALE pcigale.ini file----
Wavelength range in microns where to compute the AGN fraction. Note
that it includes all the components, not just dust emission. To use
the the total dust luminosity set to 0/0.
lambda_fracAGN = 0/0

**Comment by Guang91** [comment]
Hi @Cristol-Chang, the choice is rather arbitrary. It's the entire wavelength range where the galaxy dust module is defined. This range is wide, and should contain most of the dust emission. Of course, you can always set lambda_fracAGN to define any range you prefer.

**Reply by W-Chang-astro** [reply]
Hi Guang,

Thanks for your reply! I included the radio module when I did SED fitting through CIGALE. Does the default estimate f_agn also include the AGN-radio emission?  Or do I have to set lambda_fracAGN to cover the radio band?

**Comment by Guang91** [comment]
Good question. I'm actually not sure about it. @mboquien What do you think?
